crates/tests/src/e2e/setup.rs

```diff
@@ -1123,8 +1139,8 @@ where
 
     // Root cargo workspace manifest path
     let (bin_name, log_level) = match bin {
-        Bin::Namada => ("namada", "info"),
-        Bin::Node => ("namadan", "info"),
+        Bin::Namada => ("namada", "debug"),
+        Bin::Node => ("namadan", "debug"),
```

Comment:
can we switch this back to info? Debug blows up the artifacts in ci

Reply:
ah good catch! didn't intend to push this!